dotcom-rendering/src/web/browser/islands/getIslandsByName.ts

```diff
@@ -0,0 +1,8 @@
+export const getIslandsByName = (name: string): HTMLElement[] => {
```

Comment:
I added this under the `web/browser/islands` code because it adds a link in our codebase to gu-islands exist, and putting it here makes things easier if we migrate away from islands one day.